tdms/include/tensor_init.h

```diff
@@ -1,20 +0,0 @@
-/**
- * A collection of functions for tensor initialisation given matlab pointers
- */
-#pragma once
-#include <string>
-#include <vector>
-#include "vector_collection.h"
-#include "mat_io.h"
-#include "field.h"
-
-
-/**
- * Initialise the grid tensors/arrays, including the electric and magnetic split fields and the
- * materials array.
- * @param ptr Pointer to the matlab struct
- * @param E_s Electric split field
- * @param H_s Magnetic split field
- * @param materials Materials array
- */
```

Comment:
best to move this to https://github.com/UCL/TDMS/blob/unused-tensor-init/tdms/include/array_init.h ?